iodata/utils.py

```diff
@@ -166,3 +166,28 @@ def check_dm(dm: np.ndarray, overlap: np.ndarray, eps: float = 1e-4, occ_max: fl
     if occupations.max() > occ_max + eps:
         raise ValueError('The density matrix has eigenvalues considerably larger than '
                          'max. error=%e' % (occupations.max() - 1))
+
+
+def compute_dm_1st_order(coeffs: np.ndarray, occs: np.ndarray) -> np.ndarray:
```

Comment:
Is there a reason this isn't just called compute_1dm(...)?

Reply:
or `compute_1rdm`